aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,45 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
+
+### Roadmap Updates
+
+- **Page Layouts**
+  - Updated the [Figma file](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=2891%3A21273) to reflect new Page component in Grommet. Page component coming soon to Design System site.
+- **Global Banner Notifications**
+  - Live on the [Design System site](https://design-system.hpe.design/templates/global-banner-notifications)!
+- **Page Header**
+  - Finalized gathering use cases, annotating designs from service teams and concluded what PageHeader consists of. A share out has been conducted with team and [ideations](https://www.figma.com/file/BHp0KrOcPvbnzNyshfHkd8/HPE-Page-Header) are in progress.
+  - Conducted a user study test on the Ascending Button to understand what people are calling this button and how they interact with it.
+- **Tabs**
+  - Finalized gathering use cases from stakeholders to solve for use cases where there are more than 5 tabs. A share out has been finalized and designer is currently working with developers to optimize for a solution.
+
+
+### Stay Current with the Latest Changes
+- Included Grid documentation to help developers better understand how [Grid component](https://design-system.hpe.design/components/grid#grid-properties-and-key-concepts) works and the capabilities it has.
+- Added an example of single required FormField on the [site.](https://design-system.hpe.design/templates/forms#single-required-formfield)
+- FileInput - [Trash icon](https://design-system.hpe.design/components/fileinput) got replaced with a Close icon. 
+- Wizard - Fixed template spacing issues between elements in [Figma](https://www.figma.com/file/GEhgYzx3b8MejBjg9gAxyA/Wizard?node-id=0%3A1).
+- Menu List - Menu Lists are now swappable in [Figma](https://www.figma.com/file/DyFWlzLBUtK5KH5jJhwoCB/HPE-Menu-Component?node-id=606%3A1489)
+- [Select](https://www.figma.com/file/fZJdRcNrvZ1JjdPRYHpFFU/HPE-Select-Component?node-id=0%3A1), [TextInput](https://www.figma.com/file/E40Oo2MpdRNMyVaJqfJ0Wd/HPE-Text-Input-Component?node-id=0%3A1) and [TextArea](https://www.figma.com/file/evH1LQewFPc8Y2fteywMxM/HPE-Text-Area-Component?node-id=0%3A1) - Added auto layout to fix resizing and wrapping issues in Figma.
+- [Tabs](https://www.figma.com/file/Kp4dWyhUTnKIJ1Cg5CoL9o/HPE-Tabs-Component?node-id=208%3A0) - you can now swap to the selected instance without any issues.
+
+
+
+### Latest Releases
+- **Grommet v2.22.0** - Release includes enhancements including (but not limited to): NEW Page and PageContent components! Accessibility improvements to RangeInput, Menu, and Select components.
+Full details about this release are available in the [Release Notes](https://github.com/grommet/grommet/releases/tag/v2.22.0).
+- **Grommet-HPE-theme** - Release includes NEW Banner Notifications styling, NEW Page Component styling. Changed value of text-weak to pass color contrast requirements and can be used for text when a more subtle treatment is desired. text-xweak is recommended for disabled text.
```

Comment:
- **Grommet-HPE-theme** - Release includes NEW Banner Notifications styling, NEW Page Component styling. Changed value of `text-weak` to pass color contrast requirements and can be used for text when a more subtle treatment is desired. `text-xweak` is recommended for disabled text.